doc/classes/NetworkedMultiplayerCustom.xml

```diff
@@ -50,8 +51,9 @@
 			<argument index="1" name="buffer" type="PoolByteArray" />
 			<argument index="2" name="transfer_mode" type="int" />
 			<description>
-				Emitted when the local [MultiplayerAPI] generates a packet.
+				Emitted when the local [MultiplayerAPI] generates a packet (e.g. when calling [method Node.rpc]).
```

Comment:
+1 to this! Developers will be most familiar with RPC's, so this makes it a little clearer what's going on here.